ghost/core/core/frontend/helpers/tpl/content-cta.hbs

```diff
@@ -2,19 +2,19 @@
 <aside class="gh-post-upgrade-cta">
     <div class="gh-post-upgrade-cta-content" style="background-color: {{@site.accent_color}}">
         {{#has visibility="paid"}}
-            <h2>This {{#is "page"}}page{{else}}post{{/is}} is for paying subscribers only</h2>
+            <h2>{{t "This"}} {{#is "page"}}{{t "page"}}{{else}}{{t "post"}}{{/is}} {{t "is for paying subscribers only"}}</h2>
         {{/has}}
         {{#has visibility="members"}}
-            <h2>This {{#is "page"}}page{{else}}post{{/is}} is for subscribers only</h2>
+            <h2>{{t "This"}} {{#is "page"}}{{t "page"}}{{else}}{{t "post"}}{{/is}} {{t "is for subscribers only"}}</h2>
         {{/has}}
         {{#has visibility="tiers"}}
-            <h2>This {{#is "page"}}page{{else}}post{{/is}} is for subscribers on the {{tiers}} only </h2>
+            <h2>{{t "This"}} {{#is "page"}}{{t "page"}}{{else}}{{t "post"}}{{/is}} {{t "is for subscribers on the"}} {{tiers}} {{t "only"}}</h2>
```

Comment:
You've got an assumption here about sentence structure here that won't work for all languages.  It'll need to be something like:
`{{#is "page"}}{{t "This page is for paying subscribers only" {{else}} {{t "This post is for paying subscribers only"}} {{/is}}`